src/main/java/bio/terra/cli/serialization/userfacing/UFWorkspaceLight.java

```diff
@@ -111,12 +122,17 @@ public void print() {
     OUT.println("Name:              " + name);
     OUT.println("Description:       " + description);
     OUT.println("Cloud Platform:    " + cloudPlatform);
+
     if (cloudPlatform == CloudPlatform.GCP) {
       OUT.println("Google project:    " + googleProjectId);
       OUT.println(
           "Cloud console:     https://console.cloud.google.com/home/dashboard?project="
               + googleProjectId);
+    } else if (cloudPlatform == CloudPlatform.AWS) {
+      OUT.println("AWS account:       " + awsAccountId);
+      OUT.println("Cloud console:     https://" + awsAccountId + ".signin.aws.amazon.com/console/");
```

Comment:
We should omit this, the user will never have a username/password that will work with this link, instead they will get signed links for console access to individual controlled resources.

Reply:
Makes sense, removing console link, and displaying only account Id + tenant for now.